app/views/shared/_service_comparison.html.erb

```diff
@@ -45,6 +45,19 @@
                 <%= link_to t("views.shared.service_comparison.services.diy.cta"), Diy::FileYourselfController.to_path_helper, class: "button", "data-track-click": "service-comparison-diy" %>
               </div>
             </div>
+            <div class="service direct-file">
+              <div>
+                <h2 class="service__service-name">
+                  <%= t("views.shared.service_comparison.services.direct_file.service_name") %>
+                </h2>
+              </div>
+              <div class="service__description">
+                <%= t("views.shared.service_comparison.services.direct_file.service_description") %>
+              </div>
+              <div class="service__cta">
+                <%= link_to t("views.shared.service_comparison.services.direct_file.cta"), "https://directfile.irs.gov/state" ,class: "button", "data-track-click": "service-comparison-direct-file"%>
```

Comment:
Unrelated to the code review, do you know where we can see the tracking comparison? Is it in metabase?